src/bpftrace.cpp

```diff
@@ -601,6 +601,27 @@ std::unique_ptr<AttachedProbe> BPFtrace::attach_probe(Probe &probe, const BpfOrc
   return nullptr;
 }
 
+bool attach_reverse(Probe p)
```

Comment:
```suggestion
bool attach_reverse(const Probe &p)
```
Making it a const reference means we don't have to make a copy of the object each time this function is called.